static/app/views/monitors/utils/crontabAsText.tsx

```diff
@@ -0,0 +1,23 @@
+import cronstrue from 'cronstrue';
+
+import {shouldUse24Hours} from 'sentry/utils/dates';
+
+/**
+ * Display a human readible label of a crontab expression
+ */
+export function crontabAsText(crontabInput: string | null): string | null {
+  if (!crontabInput) {
+    return null;
+  }
+  let parsedSchedule: string;
+  try {
+    parsedSchedule = cronstrue.toString(crontabInput, {
+      verbose: false,
+      use24HourTimeFormat: shouldUse24Hours(),
+    });
+  } catch (_e) {
+    return null;
```

Comment:
I think we should always Sentry.captureMessage in these cases. If not else, just to know that we never silently break this, but it would be useful to see if we fail to parse some of these expressions